### ISSUE TITLE
The key "reply_markup" does not exist

### ISSUE THREAD

**Issue by Vanawy** [unreal4u/telegram-api#104]
## Which version are you using?
3.4.1

Output:
```
[2019-06-03 14:55:11] app.DEBUG: Array detected, mapping subobjects for key {"key":"reply_markup"} []
[2019-06-03 14:55:11] app.ERROR: The key "reply_markup" does not exist in the class! Maybe a recent Telegram Bot API update? In any way, please submit an issue (bug report) at https://github.com/unreal4u/telegram-api/issues with this complete log line {"object":"unreal4u\\TelegramAPI\\Telegram\\Types\\Message","data":{"inline_keyboard":[[{"text":"Продано","callback_data":"NULL:0"}]]}} []
```

## Description of problem
Found this error in logs

**Comment by unreal4u** [comment]
Hi! 

Telegram has released a new version of their API, which now includes a `reply_markup` in the Message object: 
https://core.telegram.org/bots/api#may-31-2019

This change has yet to be integrated into this library, will do so ASAP, thanks for reporting it!

**Comment by unreal4u** [comment]
Hi! I just released v3.4.3 which will take care of the above message. In the next week I'll be implementing a lot of the missing parts of the API that have been introduced in the last months.

Greetings and thanks for reporting this!
